awesome/database/migrations/20230820042135_014_turn_off_readonly_and_hidden_of_id_field.js

```diff
@@ -0,0 +1,28 @@
+/**
+ * @param { import("knex").Knex } knex
+ * @returns { Promise<void> }
+ */
+exports.up = function(knex) {
+  const idsToUpdate = [46, 47, 48, 49, 50, 51, 52, 53, 54, 55];

```

Comment:
You CANNOT use id to query here. It is not reliable. Please use `field` and `collection` to filter out the rows to update. Please refer to https://github.com/ocademy-ai/machine-learning/blob/main/awesome/database/migrations/20230815224628_013_insert_directus_fields_constrain_for_createdAt_and_updatedAt.js

Reply:
done